packages/lodestar/src/sync/regular.ts

```diff
@@ -68,6 +84,51 @@ export class RegularSync {
     }
   };
 
+  public receiveCommitteeAttestation = async (attestationSubnet: {attestation: Attestation; subnet: number}): 
+  Promise<void> => {
+    const attestation = attestationSubnet.attestation;
+    const subnet = attestationSubnet.subnet;
+    if (String(subnet) !== getAttestationSubnet(attestation)) {
+      return;
+    }
+    // Make sure this is unaggregated attestation
+    const aggregationBits = attestation.aggregationBits;
+    let count = 0;
+    for (let i = 0; i < aggregationBits.bitLength; i++) {
+      if (aggregationBits.getBit(i)) {
+        count++;
+      }
+    }
+    if (count !== 1) {
+      return;
+    }
+    if (await this.db.block.isBadBlock(attestation.data.beaconBlockRoot)) {
+      return;
+    }
+    const state = await this.db.state.getLatest();
+    const currentSlot = getCurrentSlot(this.config, state.genesisTime);
+    if (!(attestation.data.slot + ATTESTATION_PROPAGATION_SLOT_RANGE >= currentSlot &&
+      currentSlot >= attestation.data.slot)) {
+      return;
+    }
+    if(!isValidIndexedAttestation(this.config, state, getIndexedAttestation(this.config, state, attestation))) {
+      return;
+    }
+    // to see if we need special process for these unaggregated attestations
+    // not in the spec atm
+    // send attestation on to other modules
+    await Promise.all([
+      this.opPool.attestations.receive(attestation),
+      this.chain.receiveAttestation(attestation),
+    ]);
+  };
+
+  public receiveAggregateAndProof = async (aggregation: AggregateAndProof): Promise<void> => {
+    // a place holder for mainnet
+    // this is for aggregated attestattions
+    this.logger.debug(aggregation);
```

Comment:
This is not for mainnet, this is mandatory for 0.9.2, add Bucket in database to store those